docs/processing_information.md

```diff
@@ -56,6 +56,19 @@ Only cells that pass this FDR threshold are included in the filtered counts matr
 For some libraries, `DropletUtils::emptyDropsCellRanger()` may fail due to low numbers of droplets with reads or other violations of its assumptions.
 For these libraries, only droplets containing at least 100 UMI are included in the filtered counts matrix.
 
+### Processed gene expression data
+
+In addition to the raw gene expression data, we also provide a `_processed.rds` file containing a `SingleCellExperiment` object with the filtered, normalized counts matrix and embeddings from dimensionality reduction.
```

Comment:
Some suggested edits here. (I don't love "embeddings" as I think it might be confusing, so I went with something much more generic).
```suggestion
In addition to the raw gene expression data, we also provide a `_processed.rds` file containing a `SingleCellExperiment` object with further filtering applied, a normalized counts matrix, and results from dimensionality reduction.
```